src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/buildMultiTargeting/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The `buildMultiTargeting` should be moved up a level. It won't land in the right place in the nupkg because this is under `build`.